content/en/security_platform/guide/how-appsec-works.md

```diff
@@ -17,7 +17,7 @@ For Datadog Application Security to be compatible with your Datadog configuratio
 
 ### Performance
 
-Datadog Application Security uses processes already contained in the Agent and APM, so there are no performance implications when using it. When APM is enabled, the Datadog Library generates distributed traces. Datadog Application Security flags security activity in traces by using known attack patterns. Correlation between the attack patterns and the execution context provided by the distributed trace triggers security signals based on detection rules.
+Datadog Application Security uses processes already contained in the Agent and APM, so there are negligible performance implications when using it. When APM is enabled, the Datadog Library generates distributed traces. Datadog Application Security flags security activity in traces by using known attack patterns. Correlation between the attack patterns and the execution context provided by the distributed trace triggers security signals based on detection rules.
```

Comment:
Maybe use `low` instead of `negligible`, as depending on their setup, it could be non-negligible.

Reply:
Thanks for your suggestion. We'll keep negligible for now.